OpenSim/Simulation/Model/Actuator.h

```diff
@@ -156,7 +156,8 @@ class OSIMSIMULATION_API ScalarActuator : public Actuator {
 
     // default destructor, copy constructor, copy assignment
 
-    /** Convenience method to set controls given scalar (double) valued control */
+    // Convenience method to set controls given scalar (double) valued control 
+    // TODO remove
     //virtual void setControl(const SimTK::State &s, double control) const;
```

Comment:
Chris, can you just delete this commented-out method altogether? Then we can merge this.